app/src/main/java/com/project/symptoms/util/NotificationWrapper.java

```diff
@@ -100,17 +101,17 @@ public void startReminderFor(long symptomId){
                 .setInitialDelay(frequency, getCurrentTimeUnit())
                 .build();
         WorkManager.getInstance(context).enqueue(oneTimeWorkRequest);
+        Log.i("#", "reminder set for id "+symptomId+" in "+getCurrentFrequency());
     }
 
-    public void showReminderSetToast() {
+    public String getReminderSetMessage() {
         int frequency = getCurrentFrequency();
         String format = context.getString(R.string.reminder_set_format);
-        String text = String.format(format, frequency);
-        Toast.makeText(context, text, Toast.LENGTH_LONG).show();
```

Comment:
now this class won't show the toast but will build the message for the activity to show it appropiately